crates/libcontainer/src/syscall/linux.rs

```diff
@@ -489,13 +490,7 @@ impl Syscall for LinuxSyscall {
         flags: MsFlags,
         data: Option<&str>,
     ) -> Result<()> {
-        mount(source, target, fstype, flags, data).map_err(|err| {
-            tracing::error!(
```

Comment:
Is there any need to bring this tracing to the callers?

Reply:
Yes, 47bc85926f55c27cefd1925df7622a0b51910d75